x-pack/plugins/cases/server/client/alerts/get.ts

```diff
@@ -5,9 +5,15 @@
  * 2.0.
  */
 
+import { MgetResponseItem, GetGetResult } from '@elastic/elasticsearch/lib/api/types';
 import { CasesClientGetAlertsResponse } from './types';
 import { CasesClientArgs } from '..';
 import { AlertInfo } from '../../common/types';
+import { Alert } from '../../services/alerts';
+
+function isAlert(doc?: MgetResponseItem<unknown>): doc is GetGetResult<Alert> {
+  return Boolean(doc && !('error' in doc));
```

Comment:
Do we also need to check if `_source` exists or is that implied if `error` does not exist in the doc?